internal/commands/project.go

```diff
@@ -248,7 +248,7 @@ func runCreateProjectCommand(
 		if err != nil {
 			return err
 		}
-		groups, err := updateGroupValues(&input, cmd, groupsWrapper, featureFlagsWrapper)
```

Comment:
we need to make sure we are doing the same on project creation in scan command as well

Reply:
done